bciers/apps/registration/app/components/operations/registration/OperationInformationPage.tsx

```diff
@@ -46,8 +46,14 @@ export const createOperationInformationSchema = (
       purpose !== RegistrationPurposes.ELECTRICITY_IMPORT_OPERATION &&
       purpose !== RegistrationPurposes.POTENTIAL_REPORTING_OPERATION;
 
+    const required = isRegulatedProducts
+      ? {
+          required: ["registration_purpose"],
+        }
+      : {};
+
     return {
-      required: [isRegulatedProducts && "regulated_products"],
+      ...required,
```

Comment:
I was split on this solution or just setting `required` to `required: [isRegulatedProducts ? "regulated_products" : ""]` seemed to work. Though I opted for leaving the required field out altogether if not needed.